### DISCUSSION TITLE
Managing docker containers

### DISCUSSION THREAD

**Discussion by cowcumber** [DioxusLabs/dioxus#3388] [General]
Hello,

I tried to build a `dioxus` (fullstack) app that manages docker containers (deploy, show_info, pull_images, ..). To do so, I added the `bollard` crate into the app, but the app fails to compile after that. I've tried also the simpler 'docker' crate but nothing. I added them in both `server/Cargo.toml` and `web/Cargo.toml` (first in the former, then in the latter, exclusively).
I would like to know if this behavior is normal or if I've made something wrong. And in any case, do you know a way to achieve what i want? Thank you in advance!

**Comment by matous-volf** [comment]
I'm not one hundred percent sure, but [this docs section](https://dioxuslabs.com/learn/0.6/guides/fullstack/managing_dependencies) might help you. You could try only building the the Docker crates on the server side.
